controller.go

```diff
@@ -1042,10 +1043,15 @@ func (c *controller) NewSandbox(containerID string, options ...SandboxOption) (S
 	}
 	c.Unlock()
 
+	sandboxID := stringid.GenerateRandomID()
```

Comment:
If this is a temporary fix, can you please add a comment mentioning that is temporary and that the final fix is tracked in xxx ticket?

Reply:
This is the fix for current windows platform support and not temporary.
Once we get underlying platform support (No ETA), will update this code.